samples/unit-tests/plotbandslines/plotbandslines/demo.js

```diff
@@ -81,8 +88,18 @@ QUnit.test('Plot band labels', function (assert) {
         'Highcharts - after'
     );
 
+    const line = chart.xAxis[0].plotLinesAndBands[3].svgElem.getBBox(),
+        label = chart.xAxis[0].plotLinesAndBands[3].label.getBBox();
+
+    assert.close(
+        line.x,
+        label.x + chart.plotLeft,
+        3,
```

Comment:
Test is failing for me, maybe the tolerance should be larger?

```sh
Firefox 123.0 (Mac OS 10.15) Highcharts Plot band labels FAILED
        HTML label should be placed (x-pos) near the plot line (#20792).
        Expected: 154
        Actual: 148.5
        @samples/unit-tests/plotbandslines/plotbandslines/demo.js:94:12
        runTest@node_modules/qunit/qunit/qunit.js:2526:35
        run@node_modules/qunit/qunit/qunit.js:2509:16
        queue/runTest/<@node_modules/qunit/qunit/qunit.js:2780:16
        processTaskQueue@node_modules/qunit/qunit/qunit.js:3937:30
        processTaskQueue/<@node_modules/qunit/qunit/qunit.js:3941:23
```

Reply:
I'm still not sure if that's a good way of testing this feature. I'll try to figure out some different tests.